src/elements/content-sidebar/SidebarNavButton.js

```diff
@@ -53,6 +54,7 @@ const SidebarNavButton = ({
                                 state: sidebarState,
                             }}
                             type="button"
+                            tabIndex={isActive() ? '0' : '-1'}
```

Comment:
Can we pull the return value of `isActive()` into its own variable to avoid calling it twice?